src/main/scala/com/dwolla/cloudflare/DnsRecordClient.scala

```diff
@@ -84,16 +82,16 @@ class DnsRecordClient(executor: CloudflareApiExecutor)(implicit val ec: Executio
     }
   }
 
+  private def domainNameToZoneName(name: String): String = name.split('.').takeRight(2).mkString(".")
+
   implicit def httpResponseToJsonInput(httpResponse: HttpResponse): MonadicJValue = parseJson(httpResponse.getEntity.getContent)
 }
 
 case class MultipleCloudflareRecordsExistForDomainNameException(domainName: String, records: Set[DnsRecordDTO]) extends RuntimeException(
   s"""Multiple DNS records exist for domain name $domainName:
      |
      | - ${records.mkString("\n - ")}
-     |
-     |This resource refuses to process multiple records because the intention is not clear.
```

Comment:
I was under the impression this line was AWS Lambda specific. I can add it back if that's not the case.